open-sphere-plugins/csv-common/src/main/java/io/opensphere/csvcommon/ui/controller/ImportableColumnController.java

```diff
@@ -46,6 +49,22 @@ public void detectNonImportableColumns(CSVParseParameters parameters, Preference
 
                 List<String> columnsToNotImport = CSVColumnPrefsUtil.getCustomKeys(registry, key);
 
+                List<?> currentParams = new ArrayList<String>();
+                currentParams = parameters.getColumnNames();
+                List<String> newParams = new ArrayList<String>();
+
+                Iterator<?> litr = currentParams.iterator();
+                while (litr.hasNext())
+                {
+                    Object element = litr.next();
+                    if (element.equals("Index") || element.equals("Color"))

```

Comment:
Im not sure harding the column name is a good idea in this case. Ill have to take look at the ticket

Reply:
replaced with checking all metacolumn cases